charts/kueue/Chart.yaml

```diff
@@ -18,4 +18,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "0.4.2"
+appVersion: "v0.4.2"
```

Comment:
I changed the version to v0.4.2 because that's the format of the kueue container tags.